src/scripts/setup.sh

```diff
@@ -6,6 +6,10 @@ if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
 export GITHUB_TOKEN=${!PARAM_GH_TOKEN}
 [ -z "$GITHUB_TOKEN" ] && echo "A GitHub token must be supplied. Check the \"token\" parameter." && exit 1
 echo "export GITHUB_TOKEN=\"${GITHUB_TOKEN}\"" >> "$BASH_ENV"
+# Get hostname
+export GITHUB_HOSTNAME=${!PARAM_GH_HOSTNAME}
+[ -z "$GITHUB_HOSTNAME" ] && echo "A GitHub token must be supplied. Check the \"token\" parameter." && exit 1
```

Comment:
I think this was changed from "token" to "hostname", or the error supplied with it at least refers to the token